### ISSUE TITLE
Pix2Struct: The provided lr scheduler `LambdaLR` doesn't follow PyTorch's LRScheduler API

### ISSUE THREAD

**Issue by JakeMSc** [NielsRogge/Transformers-Tutorials#291]
I executed the [Pix2Struct notebook](https://github.com/NielsRogge/Transformers-Tutorials/blob/master/Pix2Struct/Fine_tune_Pix2Struct_on_key_value_pair_dataset_(PyTorch_Lightning).ipynb) as is, and then got this error:

```MisconfigurationException: The provided lr scheduler `LambdaLR` doesn't follow PyTorch's LRScheduler API.
You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a
custom LR scheduler.```

When running `trainer.fit(pl_module)`
